CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -184,15 +179,18 @@ class SchedulingPlacesViewController: UIViewController {
         }
 
         nextButton.snp.makeConstraints { make in
-            make.size.equalTo(nextButtonSize)
+            make.bottom.equalTo(view.safeAreaLayoutGuide).offset(-bottomPadding)
```

Comment:
you can do `.inset(bottomPadding)` here (https://github.com/SnapKit/SnapKit/issues/262)